docs/deployment-guide.md

```diff
@@ -607,20 +612,41 @@ the `configuration_files/alfresco-global.properties` file.
 
 ### Enable SSL
 
-If you have a FQDN and a certificate you want to use place the certificate and the key in the `configuration_files/ssl_certificates` folder before running the playbook. Also replace the `acs_play_fqdn_alfresco: "your_domain.com"` with your own domain in `playbooks/group_vars/all.yml` along with setting `acs_play_use_ssl: true`.
+If you have a FQDN and a certificate you want to use place the certificate and
```

Comment:
I think the phrasing is a bit odd. We should not really mention "If you have a fqdn'...
Something more like bellow would be more clear imho
> If you want to deploy Alfresco platform secured with an SSL certificate you provide, place it in....
> And make sure the domain associated with this certificate comes first in the list of `acs_play_known_urls`...